### ISSUE TITLE
ui.table Context menu actions (`.context_menu`)

### ISSUE THREAD

**Issue by mofojed** [deephaven/deephaven-plugins#321]
Dependent on #322 
Once that's done, this should be fairly straightforward.

**Comment by mofojed** [comment]
Need to nail down the syntax (should be a `prop` instead of immutable fluent interface as it's currently defined, as per Don: https://github.com/deephaven/deephaven-plugins/blob/main/plugins/ui/DESIGN.md#context_menu)

Focus on just defining menu items for now, asynchronously building the menu using a callback can be done as a separate ticket after #322 is done.
